app/views/layouts/_admin_sidebar_index.html.haml

```diff
@@ -32,3 +32,7 @@
       = link_to(admin_revision_history_path) do
         %span.fa.fa-history
         Revision History
+    %li
+      = link_to(admin_organizations_path) do
+        %span.fa.fa-group
+        Organizations
```

Comment:
Are we planning to merge this PR after the organizations are introduced or wait until they are actually useful ( custom domain and rest of the features ) ? If we are not merging it until then, then there shouldn't be any need to hide these, right?